README.md

```diff
@@ -23,21 +23,36 @@ Current reviewers are listed in [`bioimageio_collection_config.json`][review-con
    * request changes on resource drafts
    * upload a new version for any resource
 
-### Review Process
+### Adding a bioimage.io resource
 
-The review process technically starts after a user uploaded a _resource package_.
-Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_, or an updated version of any existing resource.
+A bioimage.io resource is created by a user uploading a _resource package_.
+Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_.
 Typically, uploaders would go via [bioimage.io/upload][upload].
 Alternatively, any direct link to a downloadable resource package (`.zip`-file) would work.
 The latter option is reserved for members of this repository (or the bioimageio org).
+Once available online the _resource package_ is staged (see [Staging section](#staging)), tested (see [Testing section](#testing)), and reviewed (see [Review section](#review)).
+
+
+graph TD;
+    stage[stage]
+    test[test]
+    cr[request changes]
+    publish[accept/publish]
+    backup[backup to Zenodo]
+
+    stage-->test
+    test-->cr
+    cr-->stage
+    test-->publish
+    publish-->backup
 
 #### Staging
 
-Now the stage workflow needs to be dispatched.
-If the resource package was uploaded via the bioimage.io website, this is initiated automatically.
-In case of a url to a resource package, the `stage` workflow needs to be [dispatched manually, or via github api][staging-w] ("run workflow")[^1].
+Given a download URL to a _resource package_ the `stage` workflow needs to be [dispatched manually, or via github api][staging-wf] ("run workflow")[^1].
+If the _resource package_ was uploaded via the bioimage.io website, this is initiated automatically by the uploader service.
```

Comment:
I'd have to common case first. I'd hope that would be via the website... but who knows.

Reply:
change the order